VueJS/index.html

```diff
@@ -1,8 +1,24 @@
 <!-- development version, includes helpful console warnings -->
 <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
-
+<h4>Hello Todo App</h4>
 <div id="app">
-<!-- render your list here -->
+    <div v-for="(todo,index) in todos" :row="todo.id" >
+        <ul>
+            <li v-if="editmode[todo.id]"><input key="message-update" v-model="tempValues[index].message"  class="input"/>
```

Comment:
```suggestion
            <li v-if="editmode[todo.id]">
            <input key="message-update" v-model="tempValues[index].message"  class="input"/>
```
Better wrap your elements consistently for better readability.